InvoiceMaker/InvoiceMaker/Views/WorkDone/Create.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@model InvoiceMaker.FormModels.CreateWorkDone
+
+<h1>Create a client</h1>
+
+@*@Html.DropDownList(Model.Type, new SelectList())*@
+
+
+
+@using (Html.BeginForm("Create", "WorkDone", FormMethod.Post, new { @class = "pure-form pure-form-aligned" }))
+{
+    @Html.AntiForgeryToken()
+
+
+    <fieldset>
+        <div>
+            @using (Html.BeginForm())
+            {
+                @Html.DropDownListFor(m => m.ClientId, Model.ClientSelectList, "Client")
+                @Html.ValidationMessageFor(model => model.ClientId, "", new { @class = "pure-form-message-inline" })
+
+                @Html.DropDownListFor(m => m.WorkTypeId, Model.WorkTypeSelectList, "Work Type")
+                @*<button type="submit">Submit</button>*@
+            }
+
+        </div>
+        @*<div class="pure-control-group">
+            @Html.LabelFor(model => model.Name)
+            @Html.EditorFor(model => model.Name)
+            @Html.ValidationMessageFor(model => model.Name, "", new { @class = "pure-form-message-inline" })
+        </div>*@
+
+        
+        <div class="pure-controls">
+            @Html.ActionLink("WorkDone", "Index", new { }, new { @class = "pure-button" })
```

Comment:
What is the intention of this link?

Reply:
This is to redirect the user to the index page which list's all the workdones created after he's created one